src/main/scala/com/github/unchama/seichiassist/subsystems/buildcount/subsystems/notification/bukkit/actions/BukkitNotifyBuildAmountThreshold.scala

```diff
@@ -15,6 +15,7 @@ import org.bukkit.entity.Player
 object BukkitNotifyBuildAmountThreshold {
 
   import PlayerSendable.forString
+  import cats.implicits.catsSyntaxOption
```

Comment:
nits: 実は

```suggestion
  import cats.implicits._
```

でも、`implicits` package object が全ての extension method や型クラスインスタンスなどを含んでくれているので大丈夫です。

アプリケーションコードでcatsのsyntaxを利用したい時は、特に理由がなければ `implicits._` を import するのが良いかなと思います(どの syntax object を import すればいいか迷わないで良いので)

Reply:
なるほど、`cats.implicits`以下のimportはワイルドカードでのimportがお勧めなのですね。
今回は使用するメソッドが明確だったのでimportをIDE任せにしたらこうなりました。次回以降は意識してimportしたいと思います。